ibmcloud-powervs/README.md

```diff
@@ -0,0 +1,76 @@
+# Setup instructions
+## Prerequisites
+- golang
+- pvsadm tool
+- IBM Cloud account
+
+#### Intall pvsadm
+
+We will be using the tool `pvsadm` to customise and build the VM image. Download or install it using the below instructions.
+
+1. You can download the binary from Github releases [here](https://github.com/ppc64le-cloud/pvsadm/releases)
+   
+2. To install from source
+   ```
+   cd $GOPATH/src/github.com/
+   git clone https://github.com/ppc64le-cloud/pvsadm.git
+   cd pvsadm && make build
+   ```
+
+3. For MacOS, refer [here](https://github.com/ppc64le-cloud/pvsadm#homebrew)
+
+## Image Build
+
+1. Customise the image preparation
+```
+pvsadm image qcow2ova --prep-template-default > image-prep.template
+```
+
+Add the following snippet to `image-prep.template`
+```
+yum install -y gcc gcc-c++ git make
+git clone -b CCv0 https://github.com/kata-containers/kata-containers.git
+git clone -b staging https://github.com/confidential-containers/cloud-api-adaptor.git
+cd cloud-api-adaptor/ibmcloud-powervs/image
+make build
+```
+
+> If you intend to use DHCP network type for creating peer pod VMs with
+> PowerVS provider, you need to additionally add this to `image-prep.template`
+> ```
+> mkdir -p /etc/cloud/cloud.cfg.d
+> cat <<EOF >> /etc/cloud/cloud.cfg.d/99-custom-networking.cfg
+> network: {config: disabled}
+> EOF
+> ```
+
+2. Download the qcow2 image and converts into ova type
+```
+pvsadm image qcow2ova --image-name <name> --image-dist centos --image-url https://cloud.centos.org/centos/8-stream/ppc64le/images/CentOS-Stream-GenericCloud-8-20220125.1.ppc64le.qcow2 --prep-template image-prep.template
```

Comment:
I picked  https://cloud.centos.org/centos/8-stream/ppc64le/images/CentOS-Stream-GenericCloud-8-latest.ppc64le.qcow2 to try and get passed this issues, but the command failed with:
```
failed while preparing the image for centos distro, err: script /setup.sh failed with exitstatus: 1, stdout: , stderr: fork/exec /setup.sh: exec format error
```
suggesting it was doing something on the wrong architecture? Any ideas?

Reply:
IIRC, using `pvsadm` to convert the qcow2 to ova type works only on ppc64le(CentOS/RHEL).

Reply:
ok, should that go into the pre-req as well?